### PR TITLE
Update README.md (fix badges)

### PR DESCRIPTION
**Pull request by juanitorduz**
Fix badges from https://github.com/pymc-labs/pymc-marketing/pull/1274

----
📚 Documentation preview 📚: https://pymc-marketing--1278.org.readthedocs.build/en/1278/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
 ----
 
-![Build](https://github.com/pymc-labs/pymc-marketing/actions/workflows/ci.yml/badge.svg)
-![Test](https://github.com/pymc-labs/pymc-marketing/actions/workflows/test.yml/badge.svg)
+![Build](https://github.com/pymc-labs/pymc-marketing/actions/workflows/test.yml/badge.svg)
+![Test](https://github.com/pymc-labs/pymc-marketing/actions/workflows/test_notebook.yml/badge.svg)
```
Comment (wd60622) [inline README.md:10]: Is there a doc build badge for "Build"

**Review by wd60622 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (juanitorduz) [reply README.md]: You mean like https://github.com/pymc-labs/pymc-marketing/pull/1278/commits/3488cc461ec6ac9fbcacdac3add7f06a8813b90b ?

File: README.md [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 ----
 
-![Build](https://github.com/pymc-labs/pymc-marketing/actions/workflows/ci.yml/badge.svg)
 ![Test](https://github.com/pymc-labs/pymc-marketing/actions/workflows/test.yml/badge.svg)
+![Test Notebook](https://github.com/pymc-labs/pymc-marketing/actions/workflows/test_notebook.yml/badge.svg)
```
Comment (wd60622) [inline README.md:10]: This is just duplicate now with the next line

Comment (juanitorduz) [reply README.md:10]: ahhh

Comment (juanitorduz) [reply README.md:10]: fixed!

**Review by wd60622 — APPROVED** [review]
Thanks for edits ☺️

**Pull request merged**